plugins/swagger-apibody/src/index.ts

```diff
@@ -0,0 +1,70 @@
+import type {
+  AmplicationPlugin,
+  CreateEntityControllerBaseParams,
+  DsgContext,
+  Events,
+} from "@amplication/code-gen-types";
+import {
+  getClassDeclarationById,
+  getClassMethodById,
+  interpolate,
+} from "./util/ast";
+import { EventNames } from "@amplication/code-gen-types";
+import { builders, namedTypes } from "ast-types";
+
+const funcMethodMap = {
+  CREATE_ENTITY_FUNCTION: "CREATE_INPUT",
+  UPDATE_ENTITY_FUNCTION: "UPDATE_INPUT"
+}
+
+class SwaggerApiBody implements AmplicationPlugin {
+  register(): Events {
+    return {
+      [EventNames.CreateEntityControllerBase]: {
+        before: this.beforeCreateControllerBase,
+      },
+    };
+  }
+
+  beforeCreateControllerBase(
+    context: DsgContext,
+    eventParams: CreateEntityControllerBaseParams
+  ) {
+    const { templateMapping, template, controllerBaseId } = eventParams;
+
+    interpolate(template, templateMapping);
+
+    const classDeclaration = getClassDeclarationById(
+      template,
+      controllerBaseId
+    );
```

Comment:
@barshimi I dont think you need to interpolate the template and the params 

You should be able to work directly on the template without interpolation